src/derivable/derivation.ts

```diff
@@ -127,7 +122,7 @@ export class Derivation<V> extends BaseDerivation<V> implements Derivable<V> {
         /**
          * The deriver function that is used to calculate the value of this derivation.
          */
-        private readonly _deriver: (...args: any[]) => State<V>,
+        private readonly _deriver: (this: Derivable<V>, ...args: any[]) => State<V>,
```

Comment:
There should be specs to show that these functions are now called with the original `Derivable` as `this`